test/.cache/v/cache/lastfailed

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
remove this file, try not to use add wildcard, there are not too many files add them one at a time.